packages/components/src/List/ListComposition/Manager/ListManager.component.js

```diff
@@ -5,18 +5,18 @@ import { withTranslation } from 'react-i18next';
 import { ListContext } from '../context';
 import getDefaultT from '../../../translate';
 import I18N_DOMAIN_COMPONENTS from '../../../constants';
-import { sortCollection } from './sort';
+import useCollectionSort from './sort.hook';
 import { filterCollectionByText } from './filter';
 
 function Manager(props) {
+	let collection;
+
 	const [displayMode, setDisplayMode] = useState();
-	const [sortParams, setSortParams] = useState({});
 	const [textFilter, setTextFilter] = useState();
 
-	let collection = props.collection;
-
 	// Sort items
-	collection = useMemo(() => sortCollection(collection, sortParams), [collection, sortParams]);
+	const { sortedCollection, sortParams, setSortParams } = useCollectionSort(props.collection);
```

Comment:
Sort parameter and the sort function passed to the useCollection are missing here ?

Reply:
No, they are not missing, there is no custom ones. This usage in ListManager is the uncontrolled one. In uncontrolled mode, we use the default.
The initial sort params are managed by SortBy component `initialValue` props. The purpose of the manager is not to manage all of that, but to init empty states, and propagate the value et setters to the children via context.